README.md

```diff
@@ -54,9 +79,82 @@ about "Cannot resolve module 'source-map-loader'" due to shortcomings in webpack
 Deployment
 ==========
 
+Configure the app by modifying the `config.json` file to the correct values:
+
+1. default_hs_url is for the home server url (could be http://your.server.ip:8008 if vector and synapse are on the same machine),
+2. default_is_url is for the identity server (could be empty but if you let it empty, email notifications and email login may not work).
```

Comment:
Technically, email login will work without an identity server, but registering with an email address or adding an email address to your account will not. I'd suggest, "This is the default server used for verifying third party identifiers like email addresses. If this is blank, registering with an email address or adding an email address to your account will not work."